frontend/javascripts/admin/task/task_list_view.tsx

```diff
@@ -237,6 +236,176 @@ function TaskListView({ initialFieldValues }: Props) {
+          <br />
+          <span title="Active Instances">
+            <ForkOutlined className="icon-margin-right" />
+            {status.active}
+          </span>
+          <br />
+          <span title="Finished Instances">
+            <CheckCircleOutlined className="icon-margin-right" />
+            {status.finished}
+          </span>
+          <br />
+          <span title="Annotation Time">
+            <ClockCircleOutlined className="icon-margin-right" />
+            {formatSeconds((task.tracingTime || 0) / 1000)}
+          </span>
+        </div>
+      ),
+      filters: [
+        {
+          text: "Has Pending Instances",
+          value: "pending",
+        },
+        {
+          text: "Has Active Instances",
+          value: "active",
+        },
+        {
+          text: "Has Finished Instances",
+          value: "finished",
+        },
+      ],
+      onFilter: (key: any, task: APITask) => task.status[key as unknown as keyof TaskStatus] > 0,
+    },
+    {
+      title: "Edit Position / Bounding Box",
+      dataIndex: "editPosition",
+      key: "editPosition",
+      width: 150,
+      render: (_, task: APITask) => (
+        <div className="nowrap">
+          {formatTuple(task.editPosition)} <br />
+          <span>{formatTuple(task.boundingBoxVec6)}</span>
+        </div>
+      ),
+    },
+    {
+      title: "Experience",
+      dataIndex: "neededExperience",
+      key: "neededExperience",
+      sorter: Utils.localeCompareBy<APITask>((task) => task.neededExperience.domain),
+      width: 250,
+      render: (neededExperience: APITask["neededExperience"]) =>
+        neededExperience.domain !== "" || neededExperience.value > 0 ? (
+          <Tag>
+            {neededExperience.domain} : {neededExperience.value}
+          </Tag>
+        ) : null,
+    },
+    {
+      title: "Creation Date",
+      dataIndex: "created",
+      key: "created",
+      width: 200,
+      sorter: Utils.compareBy<APITask>((task) => task.created),
+      render: (created: APITask["created"]) => <FormattedDate timestamp={created} />,
+      defaultSortOrder: "descend",
+    },
+    {
+      title: "Action",
+      key: "actions",
+      width: 170,
+      fixed: "right",
+      render: (_, task: APITask) => (
+        <>
+          {task.status.finished > 0 ? (
+            <div>
+              <a
+                href={`/annotations/CompoundTask/${task.id}`}
+                title="View all Finished Annotations"
+              >
+                <EyeOutlined className="icon-margin-right" />
+                View
+              </a>
+            </div>
+          ) : null}
+          <div>
+            <a href={`/tasks/${task.id}/edit`} title="Edit Task">
+              ,
+              <EditOutlined className="icon-margin-right" />
+              Edit
+            </a>
+          </div>
+          {task.status.pending > 0 ? (
+            <div>
+              <LinkButton onClick={_.partial(assignTaskToUser, task)}>
```

Comment:
Here is an example that causes the rendering to crash as the current code shadows the lodash variable